Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: vault
-version: 0.2.1
+version: 0.2.2
```

Comment:
Please remove this version bump, we'll bump this when we're ready to release a new version of the chart.